lib/vsc/administration/ldapsync.py

```diff
@@ -197,11 +197,15 @@ def sync_altered_groups(self, last, dry_run=True):
                 # if a 404 occured, the group is not an VO, so we skip this. Otherwise something else went wrong.
                 if err.code != 404:
                     raise
+            group_moderators = [str(m) for m in group.moderators]
```

Comment:
what's with all the `str`?

Reply:
nvm, it's just a bit odd (also `map(str, group.moderators)`, not usre it's that much more readable)

Reply:
I thought we were not in favour of `map`, and rather have list comprehensions?

Reply:
ok, np